chanzuckerberg/idseq-web
# Improve background

# Description

* Clean up the background creation and edit functions
* Fix background update bugs
* Switch background model on report page by adding background_id=<ID> 
## Type of change

- [X ] New feature (non-breaking change which adds functionality)


# Test and Reproduce
Outline the steps to test or reproduce the PR here.

1. Go to /backgrounds page, create and edit the background
2. Go to reports page like /samples/2 and add background_id =X to see the report change

## Impacted Areas in Application
List general components of the application that this PR will affect:

Which pages?


- [X] Background model change
- [X ] Report Page

Specific components 


# Checklist:

- [X ] I have run through the testing script to make sure current functionality is unchanged
- [ X] I have done relevant tests that prove my fix is effective or that my feature works
- [X ] I have spent time testing out edge cases for my feature
- [ X] I have updated the test script or pull request template if necessary
- [X ] New and existing unit tests pass locally with my changes
